packages/container/tests/container_test.js

```diff
@@ -535,6 +535,39 @@ if (isEnabled('ember-container-inject-owner')) {
       strictEqual(c, container);
     }, 'Using the injected `container` is deprecated. Please use the `getOwner` helper instead to access the owner of this object.');
   });
+
+  QUnit.test('A deprecated `container` property is appended to every object instantiated from a non-extendable factory, and a fake container is available during instantiation.', function() {
+    let registry = new Registry();
```

Comment:
Can you add `expect(5)` here (just to ensure that all the expectations are firing)?